modules/oadp-1-4-2-release-notes.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="oadp-1-4-2-release-notes_{context}"]
+= OADP 1.4.2 release notes
+
+The {oadp-first} 1.4.2 release notes lists new features, resolved issues and bugs, and known issues.
+
+[id="new-features-1-4-2_{context}"]
+== New features
+
+.Backing up different volumes in the same namespace by using the VolumePolicy feature is now possible
+
+With this release, Velero now provides resource policies to back up different volumes in the same namespace by using the `VolumePolicy` feature. The supported `VolumePolicy` feature to back up different volumes includes `skip`, `snapshot`, and `fs-backup` actions.
+link:https://issues.redhat.com/browse/OADP-1071[OADP-1071]
+
+.File system backup and data mover can now use short-term credentials
+
+File system backup and data mover can now use short-term credentials such as {aws-short} {sts-first} and {gcp-short} WIF. With this support, backup successfully completes without any `PartiallyFailed` status.
+link:https://issues.redhat.com/browse/OADP-5095[OADP-5095]
+
+[id="resolved-issues-1-4-2_{context}"]
+== Resolved issues
+
+.DPA now reports errors if VSL contains an incorrect provider value 
+
+Previously, if the provider of a Volume Snapshot Location (VSL) spec was incorrect, the Data Protection Application (DPA) reconciled successfully. With this update, DPA reports errors and requests for a valid provider value.
+link:https://issues.redhat.com/browse/OADP-5044[OADP-5044]
+
+.Data Mover restore is successful irrespective of using different OADP namespaces for backup and restore
+
+Previously, when backup operation was executed by using {oadp-short} installed in one namespace but was restored by using {oadp-short} installed in a different namespace, the Data Mover restore failed. With this update, Data Mover restore is now successful.
+link:https://issues.redhat.com/browse/OADP-5460[OADP-5460]
+
+.SSE-C backup works with the calculated MD5 of the secret key
+
+Previously, backup failed with the following error:
+[source,terminal]
+----
+Requests specifying Server Side Encryption with Customer provided keys must provide the client calculated MD5 of the secret key.
+----
+With this update, missing Server-Side Encryption with Customer-Provided Keys (SSE-C) base64 and MD5 hash are now fixed. As a result, SSE-C backup works with the calculated MD5 of the secret key. Along with this, incorrect `errorhandling` for the `customerKey` size is also fixed.
```

Comment:
```suggestion
With this update, missing Server-Side Encryption with Customer-Provided Keys (SSE-C) base64 and MD5 hash are now fixed. As a result, SSE-C backup works with the calculated MD5 of the secret key. In addition, incorrect `errorhandling` for the `customerKey` size is also fixed.
```
Rewrite to avoid a potentially ambiguous pronoun ("this" with an unclear noun reference).